web/packages/teleterm/build_resources/linux/after-install.tpl

```diff
@@ -64,4 +64,32 @@ else
   fi
 fi
 
+APPARMOR_PROFILE_SOURCE="$APP/resources/apparmor-profile"
+APPARMOR_PROFILE_TARGET="/etc/apparmor.d/teleport-connect"
+
+# Install apparmor profile.
+# First check if the version of AppArmor running on the device supports our profile.
+# This is in order to keep backwards compatibility with Ubuntu 20.04 which does not support abi/4.0.
+# In that case, we just skip installing the profile since the app runs fine without it on 20.04.
```

Comment:
Nit: I tested this on 22.04, but I think it doesn't matter here.

Reply:
Yeah oops, I meant to say 22.04.